python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -373,251 +375,246 @@ def makePlot(
+                    zoomIdx.append(idx[goodValues])
+
+            # Calculate weights for each bin to ensure that the peak of the
+            # histogram reaches 1.
+            weights = np.ones_like(values) / np.histogram(values, bins=nBinsHist)[0].max()
+
+            # Compute full-tract histogram and get its bins.
+            # NOTE: `exposure_time` histograms are quantized and look more
+            # bar-like, so they look better with fewer bins.
+            bins = ax2.hist(
+                values,
+                bins=nBinsHist,
+                label="Full Tract",
+                color=histColors[0],
+                weights=weights,
+                alpha=0.7,
+            )[1]
+
+            # Align the histogram (top right panel) with the skyproj plots.
+            pos1 = spf.ax.get_position()  # Top left.
+            pos4 = sp.ax.get_position()  # Bottom right.
+            cbarWidth = cbar.ax.get_position().height / colorBarAspect
+            # NOTE: cbarWidth != cbar.ax.get_position().width
+            ax2.set_position([pos4.x0, pos1.y0, pos4.width + cbarWidth, pos1.height])
+
+            # Overplot the histograms for the zoomed-in plots.
+            for zoomFactor, zidx, color, linestyle, hatch in zip(
+                zoomFactors, zoomIdx, histColors[1:], ["solid", "dotted"], ["//", "xxxx"]
+            ):
+                weights = np.ones_like(values[zidx]) / np.histogram(values[zidx], bins=bins)[0].max()
+                histLabel = f"{self.prettyPrintFloat(zoomFactor)}x Zoom"
+                histValues = ax2.hist(
+                    values[zidx],
+                    bins=bins,
+                    label=histLabel,
+                    color=color,
+                    weights=weights,
+                    histtype="step",
+                    linewidth=2,
+                    linestyle=linestyle,
+                    alpha=0.6,
+                )[0]
+                # Fill the area under the step.
+                ax2.fill_between(
+                    bins[:-1],
+                    histValues,
+                    step="post",
+                    color=color,
+                    alpha=0.2,
+                    hatch=hatch,
+                    label="hidden",
                 )
 
-                for line, text in zip(handles, legend.get_texts()):
-                    if isinstance(line, tuple):
-                        # Use the first handle to get the color.
-                        line = line[0]
-                    color = line.get_edgecolor() if line.get_facecolor()[-1] == 0 else line.get_facecolor()
-                    text.set_color(color)
-
-                # Add extra info to plotInfo.
-                plotInfo["nside"] = mapData.nside_sparse
-                plotInfo["valid_area"] = mapData.get_valid_area()
+            # Set labels and legend.
+            xlabel = plotInfo["property"].replace(" Map", "")
```

Comment:
_Note to myself:_ remove "Map" from the plot info title as well, i.e. go from things like:
`Per-Tract HealSparse Property Map: Sky Noise Map` to  `Per-Tract HealSparse Property Map: Sky Noise`

Also, fix this:
`Property: sky noise map` -> `Property: sky noise`